consent-ws/src/main/java/org/broadinstitute/consent/http/service/ElectionAPI.java

```diff
@@ -28,6 +28,8 @@
 
     Election describeElectionById(Integer electionId);
 
+    Election describeElectionByVoteId(Integer electionId) throws NotFoundException;
```

Comment:
i think this param should be called voteId

Reply:
@ahaessly you are right, I fixed it! Thanks